From aa6d2a0356147587f2f174e7805a63940789bb1e Mon Sep 17 00:00:00 2001
From: weaponmasterjax <42597871+weaponmasterjax@users.noreply.github.com>
Date: Sun, 6 Jul 2025 14:15:50 +0800
Subject: [PATCH 1/6] Add entries for building KSUN

Signed-off-by: weaponmasterjax <42597871+weaponmasterjax@users.noreply.github.com>
---
 drivers/Kconfig  | 1 +
 drivers/Makefile | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/drivers/Kconfig b/drivers/Kconfig
index 84a59c542405..a89157e41b0b 100644
--- a/drivers/Kconfig
+++ b/drivers/Kconfig
@@ -215,4 +215,5 @@ source "drivers/tee/Kconfig"
 
 source "drivers/mux/Kconfig"
 
+source "drivers/kernelsu/Kconfig"
 endmenu
diff --git a/drivers/Makefile b/drivers/Makefile
index ba09405ee7fc..a6391d5842c6 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -186,3 +186,5 @@ obj-$(CONFIG_TEE)		+= tee/
 obj-$(CONFIG_MULTIPLEXER)	+= mux/
 obj-$(CONFIG_TRUSTY)	+= trusty/
 obj-$(CONFIG_GNSS)		+= gnss/
+
+obj-$(CONFIG_KSU) += kernelsu/
-- 
2.52.0

