From 12aac428393be8e54c40d1cb80d65413e8966e76 Mon Sep 17 00:00:00 2001
From: Addster09 <aadityasingh.cs9@gmail.com>
Date: Wed, 12 Nov 2025 20:35:10 +0530
Subject: [PATCH 5/6] build.sh: Set "INCLUDE_KSU" to true

---
 build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.sh b/build.sh
index b7ce421089d8..5231517fe3c7 100755
--- a/build.sh
+++ b/build.sh
@@ -47,7 +47,7 @@ echo
 
 # Process options
 CLEAN_BUILD=false
-INCLUDE_KSU=false
+INCLUDE_KSU=true
 
 # Perform clean build if specified
 [ "$CLEAN_BUILD" = true ] && rm -rf out
-- 
2.52.0

